app/assets/javascripts/components/top_panel.js.jsx

```diff
@@ -16,8 +17,17 @@ var TopPanel = React.createClass({
     });
   },
 
+  updateWeek: function () {
+    this.setState({
+      next_week: !this.state.next_week
+    });
+  },
+
   submitForm: function () {
-    this.props.updateExpense( this.state.value, this.props.currentDisplay );
+    this.props.updateExpense( this.state.value, this.state.next_week, this.props.currentDisplay );
+    if (this.state.value && this.state.next_week) {
+      Materialize.toast('Added to Next Week', 2000);
```

Comment:
I like this, can you post a gif of how it looks? or screenshot. Also would be nice if this toast also showed the amount